website/docs/docs/build/python-models.md

```diff
@@ -221,6 +241,29 @@ def model(dbt, session):
 
 </File>
 
+<VersionBlock firstVersion="1.8">
+    
+
+#### Dynamic configurations
```

Comment:
hmm, this H4 isn't being rendered properly in preview: 

![Screenshot 2024-02-22 at 8 42 09 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/58d85975-0fbb-4c98-807d-ab8361b289ce)